### ISSUE TITLE
binary mode doesn't take an encoding argument

### ISSUE THREAD

**Issue by jextrevor** [ukdtom/ExportTools.bundle#144]
I'm trying to export a music playlist from my Plex Media Server running on Arch Linux. I've configured the path and ensured that the folder it's trying to export into exists. When I look in the logs, I see this:

```

2020-09-13 17:56:20,855 (7f55ff7fe640) :  DEBUG (__init__:1203) - ******* Starting scanPList with an URL of: http://127.0.0.1:32400/playlists/3829/items
2020-09-13 17:56:20,856 (7f55ff7fe640) :  DEBUG (networking:143) - Requesting 'http://127.0.0.1:32400/playlists/3829/items?X-Plex-Container-Start=0&X-Plex-Container-Size=0'
2020-09-13 17:56:20,910 (7f55ff7fe640) :  DEBUG (__init__:1214) - Writing headers for Playlist Export
2020-09-13 17:56:20,911 (7f55ff7fe640) :  CRITICAL (__init__:1242) - Detected an exception in scanPList
2020-09-13 17:56:20,911 (7f55ff7fe640) :  CRITICAL (__init__:828) - Exception happened in backgroundScanThread was binary mode doesn't take an encoding argument (most recent call last):
  File "/var/lib/plex/Plex Media Server/Plug-ins/ExportTools.bundle/Contents/Code/__init__.py", line 816, in backgroundScanThread
    scanPhotoDB(myMediaURL, outFile, level=myLevel)
  File "/var/lib/plex/Plex Media Server/Plug-ins/ExportTools.bundle/Contents/Code/__init__.py", line 1215, in scanPList
    output.createHeader(outFile, 'playlist', playListType, level=level)
  File "/var/lib/plex/Plex Media Server/Plug-ins/ExportTools.bundle/Contents/Code/output.py", line 273, in createHeader
    wrap.set_text_wrap()
ValueError: binary mode doesn't take an encoding argument

2020-09-13 17:56:20,912 (7f55ff7fe640) :  CRITICAL (runtime:1128) - Exception in thread named 'backgroundScanThread' (most recent call last):
  File "/usr/lib/plexmediaserver/Resources/Plug-ins-a78fef9a9/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/runtime.py", line 1126, in _start_thread
    f(*args, **kwargs)
  File "/var/lib/plex/Plex Media Server/Plug-ins/ExportTools.bundle/Contents/Code/__init__.py", line 816, in backgroundScanThread
    scanPhotoDB(myMediaURL, outFile, level=myLevel)
  File "/var/lib/plex/Plex Media Server/Plug-ins/ExportTools.bundle/Contents/Code/__init__.py", line 1215, in scanPList
    output.createHeader(outFile, 'playlist', playListType, level=level)
  File "/var/lib/plex/Plex Media Server/Plug-ins/ExportTools.bundle/Contents/Code/output.py", line 273, in createHeader
    wrap.set_text_wrap()
ValueError: binary mode doesn't take an encoding argument

```

The Plex Media Server version is 1.20.1.3252 and the plugin version is 2.0.0.12.

Please let me know if any additional info would be helpful.

**Comment by ukdtom** [comment]
I need all ET logs in a zip, and not just a snip

**Comment by ukdtom** [comment]
Ping

**Comment by jextrevor** [comment]
[logs.zip](https://github.com/ukdtom/ExportTools.bundle/files/5248591/logs.zip)

**Comment by pugfantus** [comment]
Any update on this?  I'm getting the same error on one of my libraries as well.

**Comment by ukdtom** [comment]
Nope, completely sidetracked with real work, so hobby sadly has to wait :(

**Comment by ukdtom** [comment]
Fix will be in V2.0.0.14 when released
